teos/cli/teos_cli.py

```diff
@@ -14,37 +15,65 @@
 from common.exceptions import InvalidKey, InvalidParameter, SignatureError, TowerResponseError
 
 from teos import DEFAULT_CONF, DATA_DIR, CONF_FILE_NAME
-from teos.cli.help import show_usage, help_get_all_appointments
+from teos.cli.help import (
+    show_usage,
+    help_get_all_appointments,
+    help_get_tower_info,
+    help_get_users,
+    help_get_user,
+)
 from teos.protobuf.tower_services_pb2_grpc import TowerServicesStub
+from teos.protobuf.user_pb2 import GetUserRequest
 
 
-def get_all_appointments(rpc_host, rpc_port):
+def to_json(obj):
+    return json.dumps(obj, indent=4)
+
+
+def formatted(func):
```

Comment:
This does not seem to fit here, specially given it could be reused for the client or other modules that need pretty data to be shown. `common.tools` may be a better fit.

Also, this has no checks at all, maybe make sure that at least the input is a string so it can be dumped as json. Also add docs.

Reply:
I'm not sure this is (or can be made) generic enough to put it in a common module: the wrapped function must return something that can be passed to `json_format.MessageToDict` (e.g. e whole Message response, or some subfields), and the output dict is then passed to `to_json`, which is just a custom wrapper for `json.dumps` to make sure that the conversion to json is done in the same way for other places that don't user `formatted` (currently only `RPCClient.get_users`).

Reply:
Ok, added a comment in 4e14ad64b379327f65fc67724f18f182777e2dfb to clarify the purpose of to_json.